articles/howtos/make-template-decorator-ready.md

```diff
@@ -0,0 +1,120 @@
+---
+uid: MakeTemplateDecoratorReady
+---
+# How do I make a Template, Decorator Ready?
+
+Click [here](xref:Decorator) to find out more about Decorators.
+
+This how-to will serve as an example of how to update a Template to allow Decorators to inject code during runtime. Please update it on your side according to what you need.
+
+Open up the Template that you wish to become more extensible. You will find that in this how-to we will use what we've done in a [previous tutorial](xref:TutorialCreateModule) to demonstrate how to achieve this. Please interpolate on your end what is written here to make the necessary adjustments on what you're trying to achieve.
```

Comment:
```suggestion
```

In line with my comments on `create-new-decorator.md`, we can consider omitting this, but further down the article we could put a TIP or something to the effect of:

```
> [!TIP]
> The [](xref:TutorialCreateModule) article describes how to create a `Module`.
```

(As per one of my earlier comments, we could leave the square brackets empty to have the article name resolved automatically.)

Reply:
Please see my comment on the related `create-new-decorator.md` section